db/aws.js

```diff
@@ -0,0 +1,33 @@
+const aws = require('aws-sdk');
```

Comment:
Replace all the 'let' with 'const'. Nowhere you are reassigning value to a variable declared with let so you can use 'const' instead.